storage/remote/codec_test.go

```diff
@@ -117,14 +131,40 @@ var (
 					HelpRef: 17, // Symbolized writeV2RequestSeries2Metadata.Help.
 					// No unit.
 				},
-				Samples:    []writev2.Sample{{Value: 2, Timestamp: 20}},
-				Exemplars:  []writev2.Exemplar{{LabelsRefs: []uint32{13, 14}, Value: 2, Timestamp: 20}},
-				Histograms: []writev2.Histogram{writev2.FromIntHistogram(30, &testHistogram), writev2.FromFloatHistogram(40, testHistogram.ToFloat(nil))},
+				Samples:   []writev2.Sample{{Value: 2, Timestamp: 20}},
+				Exemplars: []writev2.Exemplar{{LabelsRefs: []uint32{13, 14}, Value: 2, Timestamp: 20}},
+				Histograms: []writev2.Histogram{
+					writev2.FromIntHistogram(50, &testHistogram),
+					writev2.FromFloatHistogram(60, testHistogram.ToFloat(nil)),
+					writev2.FromIntHistogram(70, &testHistogramCustomBuckets),
+					writev2.FromFloatHistogram(80, testHistogramCustomBuckets.ToFloat(nil)),
+				},
 			},
 		},
 	}
 )
 
+func TestHistogramFixtureValid(t *testing.T) {
```

Comment:
note to reviewers: not related to NHCB, but adds a self check to use valid histograms for testing.